packages/client/components/Dashboard/LeftDashNavItem.tsx

```diff
@@ -26,7 +36,7 @@ const NavItem = styled(PlainButton)<{isActive: boolean}>(({isActive}) => ({
   [makeMinWidthMediaQuery(Breakpoint.SIDEBAR_LEFT)]: {
     borderRadius: '0 4px 4px 0'
   },
-  ':hover,:focus': {
+  ':hover': {
```

Comment:
This fixed the buggy hovering logic for me mentioned here: https://github.com/ParabolInc/parabol/pull/7733#pullrequestreview-1291325958